forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -90,7 +91,8 @@ internal class CapturePaymentRepository(
         contentId: String,
         paymentRef: String
     ): ForageApiResponse<String> {
-        var attempt = 1
+        var attempt = 0
+        val pollingIntervals = LDManager.getPollingIntervalsFeatureFlag() // TODO how to get the Application and logger params
```

Comment:
Don't believe you'll need to get the Application or logger params here anymore.

Reply:
forgot to delete that comment, whoops